src/useSize.tsx

```diff
@@ -67,7 +67,7 @@ const useSize = (
     }
 
     return () => {
-      if (window) {
+      if (window && window.removeEventListener) {
```

Comment:
I'm not sure I understand how this bug happens. Is `<iframe>` removed from DOM before this line executes?

Reply:
yes, I believe it could happen, such as  under `contenteditable` element,  though it is rare. I am not sure if we should hold this scene. ?

Reply:
Has it ever happened? Can you reproduce it?

Reply:
Here, I can show you how to reproduce it in a simple way by a  [demo](https://wes4r.csb.app/).

Reply:
Can you please post a screenshot, I'm on Safari 13.

Reply:
![480p-gif](https://user-images.githubusercontent.com/11720452/67649446-1e37b280-f974-11e9-8a70-f9b8e54f3b6a.gif)

on iOS 11.4 (Simulator from Xcode)